src/IceRpc.Templates/Templates/IceRpc-Protobuf-Client/Program.cs

```diff
@@ -33,10 +32,8 @@
         return customChain.Build((X509Certificate2)certificate!);
     }
 };
-#endif
```

Comment:
I merged these two consecutive `#if (transport == "quic")`, it was causing multiple empty lines.